client-java/controller/src/main/java/org/evomaster/client/java/controller/internal/db/constraint/DbTableCheckExpression.java

```diff
@@ -0,0 +1,23 @@
+package org.evomaster.client.java.controller.internal.db.constraint;
+
+/**
+ * A Sql CHECK expression on a table of the schema
+ */
+public class DbTableCheckExpression extends DbTableConstraint {
+
+    private final /*non-null*/ String sqlCheckExpression;
+
+
+    public DbTableCheckExpression(String tableName, String sqlCheckExpression) {
+        super(tableName);
+        if (sqlCheckExpression == null) {
+            throw new IllegalArgumentException("check expression cannot be null");
+        }
+        this.sqlCheckExpression = sqlCheckExpression;
+    }
```

Comment:
if you prefer, you could also write it like:
`this.sqlCheckExpression = Objects.requireNonNull(sqlCheckExpression);`
it would simplify the code quite a bit. there are many constructors in this PR in which it could used

Reply:
Ok. I will add those checks and remove the if(...) statements.